server/makefile

```diff
@@ -14,5 +14,4 @@ install:
 	@ go install ./...
 
 test:
-	@ go test -p 1 ./...
-
+	@ go test -p 1 `go list ./... | grep -v vendor`
```

Comment:
since golang v1.8  `go test` does not test vendor folder due to community requests, so it's not necessary